avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/PrepRankingData.scala

```diff
@@ -142,6 +143,27 @@ object PrepRankingData {
     }
   }
 
+  /**
+    * Based on input rank, to generate schema for the output. It's used for rank larger than 1. For rank1, it's using [[SparseVector]] directly
+    * @param rank input rank, should be larger than 1
+    * @return schema, for example, if rank == 2, the returned schema should be:
+    *         indices0: longArray
+    *         indices1: longArray
+    *         values: floatArray
+    *         if rank == 3, the returned schema should be
+    *         indices0: longArray
+    *         indices1: longArray
+    *         indices2: longArray
+    *         values: floatArray
```

Comment:
It might be more concise to say something like:
```
@return ... if rank == n, then the returned schema should be:
            indices0: longArray
            ...
            indices(n-1): longArray
            values: floatArray
```

Reply:
Fixed. Thanks